cmd/tailscale/cli/serve_test.go

```diff
@@ -54,6 +55,18 @@ func TestServeConfigMutations(t *testing.T) {
 		wantErr: exactErr(flag.ErrHelp, "flag.ErrHelp"),
 	})
 
+	// https
+	add(step{reset: true})
+	add(step{
+		command: cmd("/ proxy 3000"),
+		reset:   false,
+		want: &ipn.ServeConfig{Web: map[ipn.HostPort]*ipn.WebServerConfig{
+			"foo:443": {Handlers: map[string]*ipn.HTTPHandler{
+				"/": {Proxy: "localhost:3000"},
```

Comment:
127.0.0.1:3000 instead? I don't think "localhost" is hard-coded yet in Go and I don't want to worry (or even think) about hostname/DNS lookups in that path.

Reply:
![image](https://user-images.githubusercontent.com/79330/201550425-335334c5-ac4d-4e48-bf57-feae646358c7.png)